src/simularium/VisGeometry.ts

```diff
@@ -370,14 +371,68 @@ class VisGeometry {
         } else {
             this.resetBounds(DEFAULT_VOLUME_DIMENSIONS);
         }
-        if (this.resetCameraOnNewScene) {
-            this.resetCamera();
+
+        // Get default camera transform values from data
+        if (trajectoryFileInfo.cameraDefault) {
+            this.cameraDefault = trajectoryFileInfo.cameraDefault;
+        } else {
+            this.logger.warn(
+                "Using default camera settings since none were provided"
+            );
+            this.cameraDefault = DEFAULT_CAMERA_SPEC;
         }
+        // Reset then position and orient the camera
+        this.resetCamera();
     }
 
+    // Called when a new file is loaded, the Clear button is clicked, or Reset Camera button is clicked
     public resetCamera(): void {
         this.followObjectId = NO_AGENT;
         this.controls.reset();
+        this.positionCamera();
+    }
+
+    public positionCamera(): void {
```

Comment:
nit, but this is more `resetCameraPosition`, or `returnCameraToInitPosition`.  `positionCamera` sounds like I'm going to be able to give the function a position to set.

Reply:
I see what you're saying. Renamed to `resetCameraPosition`